docs/guides/engineering/nextjs-setup.md

```diff
@@ -9,14 +9,16 @@ This guide walks through how to setup an Eppo Feature Flag on a client and serve
 ```tsx
 npx create-next-app
 ```
+
 Provide the following answers when prompted:
+
 - What is your project named? `react-sdk-walkthrough`
 - Would you like to use TypeScript? `Yes`
 - Would you like to use ESLint? `Yes`
 - Would you like to use Tailwind CSS? `No`
 - Would you like to use `src/` directory? `No`
 - Would you like to use App Router? (recommended) `Yes`
-- Would you like to customize the default import alias (@/*)? `No`
+- Would you like to customize the default import alias (@/\*)? `No`
```

Comment:
The backslash was added automatically as an escape character by VS code formatting